README.rst

```diff
@@ -163,6 +163,7 @@ Requirements
 ============
 
 - Python >= 3.7
+- setuptools >= 46.4.0
```

Comment:
This is a list of runtime requirements, don't add build requirements here.

Reply:
Hmm, where do you think this should go in the docs?

Reply:
Honestly, I don't see any need for this to be documented because PEP 517 is standardized, hence it's well-known that the source of truth is in `pyproject.toml`.

Reply:
The only place close enough to mentioning the build process is https://docs.aiohttp.org/en/stable/contributing.html. I don't think we have any other dedicated docs.

Reply:
Ok, well I removed it.